modules/transport/PostMessageTransportBackend.js

```diff
@@ -37,6 +37,7 @@ const LEGACY_INCOMING_METHODS = [
  */
 const LEGACY_OUTGOING_METHODS = [
     'display-name-change',
+    'email-change',
```

Comment:
I don't think you need to add it here! As described in the comments this array handles the use case when apps are listening for messages directly from the iframe and they expect the old format of the message. We are still sending these messages only for backward compatibility purposes. Since this is new message I don't see why we need it here. Is there any reason that I'm missing?

Reply:
There is not any particular reason But since we are providing backward compatibility I guess there is no harm in adding this to it too.
Is it?

Reply:
I prefer to not have it there. The idea is that we don't want to break the services of the people that were using the old way but we also don't want to encourage the people to use it the old way. Please remove it from this array!